src/helpers/submitPaymentTransaction.ts

```diff
@@ -4,35 +4,60 @@ import StellarSdk, {
   MemoValue,
   Keypair,
 } from "stellar-sdk";
+// @ts-ignore
+import { AuthType } from "constants/types.d";
 import { PaymentTransactionParams } from "ducks/sendTransaction";
 import { getNetworkConfig } from "helpers/getNetworkConfig";
-import { store } from "App";
+import { trezorSignTransaction } from "helpers/wallet/trezorSignTransaction";
+import { store } from "config/store";
+
+const getSignature = (transaction: Transaction, authType: AuthType) => {
```

Comment:
Not completely sure how Trezor works but here it looks like it's signing the transaction from the "trezorSignTransaction" method. 

Is "getSignature" the right name? Or maybe "trezorSignTransaction" could be renamed to "getTrezorSignature"

Reply:
My idea here was to have `getSignature` like a helper to get signatures from all wallets (we have only Trezor there at the moment), but I will try to use `wallet-sdk` KeyManager instead, which already has Ledger signing. Most likely this will change.